### PR TITLE
Docs update for side-car adoption of faas-netes controller

### PR DESCRIPTION
**Pull request by rawkode**
https://github.com/openfaas/faas-netes/pull/191

Signed-off-by: David McKay <david.andrew.mckay@gmail.com>

### CODE REVIEW

**Comment by rawkode** [comment]
Looks like my Markdown linter cleaned up a few things also :smiley:

File: docs/deployment/troubleshooting.md [hunk 1]
```diff
@@ -212,9 +213,9 @@ $ kubectl get events --sort-by=.metadata.creationTimestamp -n openfaas-fn
 ### Check logs of the core services
 
 ```
-$ kubectl logs -n openfaas-fn deploy/faas-netes
```
Comment (alexellis) [inline docs/deployment/troubleshooting.md]: Hmm that was clearly wrong :-/

**Comment by alexellis** [comment]
Hi @rawkode thanks for the PR. Can you please update the PR in faas-netes with details of your testing? After that we can merge the PRs.

**Comment by rawkode** [comment]
I'm working on acceptance tests we can use in CI. Please bare with me

**Comment by alexellis** [comment]
Hi @rawkode this work is merged and released, so the docs can be merged now too.

Please could you rebase?

It may be worth stating the release version at which the new instructions for finding a named container logs within a Pod are needed too. Some people may have older versions deployed.

Alex

**Comment by rawkode** [comment]
@alexellis Rebased

**Pull request merged**